client/components/AllArtwork.js

```diff
@@ -33,18 +37,16 @@ export class AllArtwork extends Component {
       genres: this.props.genres
     });
   }
+
   async changeFilter(ev) {
```

Comment:
Can we add a comment on how this function works. Also renaming the function to something like filtersByCategoryFromDropDown would make it more clear, though its a long variable name haha

Reply:
this ain't really part of this PR